src/index.html

```diff
@@ -264,31 +260,6 @@ <h4 id='sleep-calendar-quality-average-weekly'></h4>
           </div>
         </section>
       </section>
-      <!-- <form class="sleep-input-form hide">
-        <input type="date" value="2019-06-15" max="2019-09-22" class="date-input input">
-        <input type="number" placeholder="hours slept" class="hours-slept-input input"  min="1" max="12">
-        <select class="sleep-quality-input input">
-  	       <option value="0">Quality of Sleep</option>
-  	       <option value="1">1</option>
-  	       <option value="2">2</option>
-  	       <option value="3">3</option>
-  	       <option value="4">4</option>
-  	       <option value="5">5</option>
-         </select>
-        <button class="submit-button">submit</button>
-      </form>
-      <form class="hydration-input-form hide">
-        <input type="date" value="2019-06-15" max="2019-09-22" class="hydration-date-input input">
-        <input type="number" placeholder="ounces drank" class="ounces-drank-input input"  min="1">
-          <button class="hydration-submit-button">submit</button>
-      </form>
-      <form class="activity-input-form hide">
-        <input type="date" value="2019-06-15" max="2019-09-22" class="activity-date-input input">
-        <input type="number" placeholder="number of steps" class="number-steps-input input"  min="1">
-        <input type="number" placeholder="minutes active" class="minutes-active-input input"  min="1">
-        <input type="number" placeholder="flights of stairs" class="stairs-input input"  min="1">
-          <button class="activity-submit-button">submit</button>
-      </form> -->
       <p class="successful-submit hide">Data Submitted Successfully!</p>
       <p class="failed-submit hide">Please Fill Out the Whole Form!</p>
       <button class="clear-button hide">Clear</button>
```

Comment:
these 3 `p`'s and the `button` should also be able to be removed (they're on the top of the file now). Otherwise I can get it on my next PR.

Reply:
I will remove it real quick